etna/models/nn/deepar_new/loss.py

```diff
@@ -0,0 +1,173 @@
+from etna import SETTINGS
+
+if SETTINGS.torch_required:
+    import torch
+    from torch.distributions import NegativeBinomial
+    from torch.distributions import Normal
+    from torch.nn.modules.loss import _Loss
+
+
+class GaussianLoss(_Loss):
+    """GaussianLoss is used to count loss in DeepAR model."""
```

Comment:
Better to say that it is negative log likelihood loss for Gaussian distribution(the same for Binomial)